### DISCUSSION TITLE
Lack of gcp-periodic mode examples

### DISCUSSION THREAD

**Discussion by lpzdvd-packlink** [cloud-custodian/cloud-custodian#7497] [GCP] [answered]
Hi,

I'm struggling to write a policy using this execution mode..
Does anyone can provide a full working example?

Thanks in advance !

**Comment by thisisshi** [comment, marked as answer]
Hi, @lpzdvd-packlink after looking into the gcp-periodic mode, it looks like there are a few issues with the mode, I've started a PR here: #7498

**Reply by lpzdvd-packlink** [reply]
Thank you for the reply. We're interested in the gcp roadmap and trying to get some examples working in each execution mode. Glad to hear that you are working on it.
